client/src/components/MultiJurisdictionAudit/index.tsx

```diff
@@ -23,11 +24,26 @@ import useRoundsAuditAdmin from './useRoundsAuditAdmin'
 import useAuditSettingsJurisdictionAdmin from './RoundManagement/useAuditSettingsJurisdictionAdmin'
 import H2Title from '../Atoms/H2Title'
 import CSVFile from './CSVForm'
+import { useInterval } from '../utilities'
+
+export const prettifyRefreshStatus = (refreshTime: number) => {
+  if (refreshTime < 10000) return 'Refreshed just now'
+  if (refreshTime < 60000)
+    return `Refreshed ${Math.floor(refreshTime / 10000) * 10} seconds ago`
+  if (refreshTime < 120000) return `Refreshed 1 minute ago`
+  return `Refreshed ${Math.floor(refreshTime / 60000)} minutes ago`
+}
 
 const VerticalInner = styled(Inner)`
   flex-direction: column;
 `
 
+const RefreshStatusWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 40px;
+`
```

Comment:
since the content inside changes length, might be nice to give this (or the tag) a fixed width. not totally sure which would look better, just a random thought

Reply:
@jonahkagan Agreed!